app/src/examples/SharedElementTransitions/Card.tsx

```diff
@@ -1,23 +1,40 @@
 import * as React from 'react';
 import { View, TouchableNativeFeedback, StyleSheet } from 'react-native';
-import { ParamListBase } from '@react-navigation/native';
 import {
   createNativeStackNavigator,
+  NativeStackNavigationProp,
   NativeStackScreenProps,
 } from '@react-navigation/native-stack';
 import Animated from 'react-native-reanimated';
 
+type ParamList = {
+  Screen1?: {};
```

Comment:
Any reason why shouldn't it be

`Screen1: object`
?

Reply:
Well, actually it should be `Record<PropertyKey, never>`.  But I agree, `object` would be better than `{}`